app-containers/podman/podman-4.7.0.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module tmpfiles linux-info
+
+DESCRIPTION="A tool for managing OCI containers and pods with Docker-compatible CLI"
+HOMEPAGE="https://github.com/containers/podman/ https://podman.io/"
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/containers/podman.git"
+else
+	SRC_URI="https://github.com/containers/podman/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64"
+fi
+LICENSE="Apache-2.0 BSD BSD-2 CC-BY-SA-4.0 ISC MIT MPL-2.0"
+SLOT="0"
+IUSE="apparmor btrfs cgroup-hybrid docker +fuse +init +rootless +seccomp selinux systemd"
+RESTRICT="test"
+
+RDEPEND="
+	app-crypt/gpgme:=
+	>=app-containers/conmon-2.0.0
+	>=app-containers/containers-common-0.56.0
+	dev-libs/libassuan:=
+	dev-libs/libgpg-error:=
+	sys-apps/shadow:=
+
+	apparmor? ( sys-libs/libapparmor )
+	btrfs? ( sys-fs/btrfs-progs )
+	cgroup-hybrid? ( >=app-containers/runc-1.0.0_rc6  )
+	!cgroup-hybrid? ( app-containers/crun )
+	docker? ( !app-containers/docker-cli )
+	fuse? ( sys-fs/fuse-overlayfs )
+	init? ( app-containers/catatonit )
+	rootless? ( app-containers/slirp4netns )
+	seccomp? ( sys-libs/libseccomp:= )
+	selinux? ( sec-policy/selinux-podman sys-libs/libselinux:= )
+	systemd? ( sys-apps/systemd:= )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-go/go-md2man
+"
+
+PATCHES=(
+	"${FILESDIR}/seccomp-toggle-4.7.0.patch"
+)
+
+CONFIG_CHECK="
+	~USER_NS
+"
+
+pkg_setup() {
+	use btrfs && CONFIG_CHECK+=" ~BTRFS_FS"
+	linux-info_pkg_setup
+}
+
+src_prepare() {
+	default
+	local file
+	for file in apparmor_tag btrfs_installed_tag btrfs_tag selinux_tag systemd_tag; do
+		[[ -f hack/"${file}".sh ]] || die
+	done
+
+	local feature
+	for feature in apparmor selinux systemd; do
+		cat <<-EOF > hack/"${feature}"_tag.sh || die
+		#!/usr/bin/env bash
+		$(usex ${feature} "echo ${feature}" echo)
+EOF
+	done
+
+	echo -e "#!/usr/bin/env bash\n echo" > tee hack/btrfs_installed_tag.sh || die
```

Comment:
The tee doesn't belong since you changed the pipe to a redirect, right?

Reply:
This is intentional. Echo nullifies hack/btrfs_installed_tag.sh. Redirect only targets hack/btrfs_tag.sh

Reply:
Well, this is the result I get, and that not intended, is it? The echo output goes into ${S}/tee and hack/btrfs_installed_tag.sh seems to be unaffected.
```console
$ sudo ebuild podman-4.7.0.ebuild clean prepare
 * podman-4.7.0.tar.gz BLAKE2B SHA512 size ;-) ...                                                                                                                   [ ok ]
 * Determining the location of the kernel source code
 * Found kernel source directory:
 *     /usr/src/linux
 * Found sources for kernel version:
 *     6.4.9-0810-x86_64
 * Checking for suitable kernel configuration options ...                                                                                                            [ ok ]
>>> Unpacking source...
>>> Unpacking podman-4.7.0.tar.gz to /var/tmp/portage/app-containers/podman-4.7.0/work
>>> Source unpacked in /var/tmp/portage/app-containers/podman-4.7.0/work
>>> Preparing source in /var/tmp/portage/app-containers/podman-4.7.0/work/podman-4.7.0 ...
 * Applying seccomp-toggle-4.7.0.patch ...                                                                                                                           [ ok ]
>>> Source prepared.
$ cat /var/tmp/portage/app-containers/podman-4.7.0/work/podman-4.7.0/hack/btrfs_installed_tag.sh
#!/usr/bin/env bash
${CPP:-${CC:-cc} -E} ${CPPFLAGS} - > /dev/null 2> /dev/null << EOF
#include <btrfs/ioctl.h>
EOF
if test $? -ne 0 ; then
	echo exclude_graphdriver_btrfs
fi
$ cat /var/tmp/portage/app-containers/podman-4.7.0/work/podman-4.7.0/tee
#!/usr/bin/env bash
 echo hack/btrfs_installed_tag.sh
```

Reply:
Oohh what a major disaster caused by tiny tee. Poor job by me at reading your comment (ADHD at worst xD). Sorry. Fixed in latest push